core/modules/field/field.crud.inc

```diff
@@ -227,9 +227,9 @@ function field_update_field($field) {
  * Validates a field configuration.
  *
  * @param $field
- *   A field structure. $field['field_name'], $field['type'] must be provided.
+ *   field['field_name'], $field['type'] must be provided.
```

Comment:
I apologize. I meant the comment should say *A field structure. $field['field_name'] **and** $field['type'] must be provided.*